packages/neuron-ui/src/components/Send/hooks.ts

```diff
@@ -10,7 +10,10 @@ import {
   shannonToCKBFormatter,
   calculateFee,
   validateOutputs,
+  DefaultLockInfo,
 } from 'utils'
+import { scriptToAddress } from '@nervosnetwork/ckb-sdk-utils'
```

Comment:
`@nervosnetwork/ckb-sdk-utils` has been migrated to `lumos`

Reply:
The `UI` package has not imported Lumos. Maybe we should remove it by the refactor that has removed`@nervosnetwork/ckb-sdk-utils`.

Reply:
https://github.com/Magickbase/neuron-public-issues/issues/201#issuecomment-1809635406